guides/SPI.md

```diff
@@ -21,6 +21,21 @@ In order to become a _service provider_, you have two options:
 * Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+## Integration with Wix
+In order to enable Wix applications to communicate with your provider application, Wix needs to know some configuration details. You add the configurations via the Dev Center by entering your app’s url and method and other configuration details in the **JSON Editor** in the service’s extension description page. 
+The **Instructions** section of the extension description page provides a list of the required configuration parameters and types. 
+
+![alt text](../media/spi-config.png)
```

Comment:
Recommend moving your screen shot to the procedure itself, to the appropriate step. When you get a real screen shot to replace this one, I suggest adding text to the JSON editor to match what's in the **Instructions** panel, so you give the reader an additional clue on usage.